app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
+import datetime
```

Comment:
According to the PEP8  Imports should be grouped in this order: standard library imports, related third-party imports, and local application/library specific imports. Separate these groups with a blank line.

Also, consider using parentheses to wrap the imports and have them all on one line.
```suggestion
import datetime

from app.errors import (
    NotVaccinatedError,
    OutdatedVaccineError,
    NotWearingMaskError
)
```